Deprecation warnings with Xcode 10.2 / Swift 5 / iOS 12

I'm getting some deprecation warnings when adding Valet as a pod dependency to my project.

```text
Valet/Sources/SecureEnclaveAccessControl.swift:79:25: warning: 'touchIDAny' was deprecated in iOS 11.3: renamed to 'SecAccessControlCreateFlags.biometryAny'
                return .touchIDAny
                        ^
Valet/Sources/SecureEnclaveAccessControl.swift:79:25: note: use 'SecAccessControlCreateFlags.biometryAny' instead
                return .touchIDAny
                        ^~~~~~~~~~
                        SecAccessControlCreateFlags.biometryAny
Valet/Sources/SecureEnclaveAccessControl.swift:86:25: warning: 'touchIDCurrentSet' was deprecated in iOS 11.3: renamed to 'SecAccessControlCreateFlags.biometryCurrentSet'
                return .touchIDCurrentSet
                        ^
Valet/Sources/SecureEnclaveAccessControl.swift:86:25: note: use 'SecAccessControlCreateFlags.biometryCurrentSet' instead
                return .touchIDCurrentSet
                        ^~~~~~~~~~~~~~~~~
                        SecAccessControlCreateFlags.biometryCurrentSet
```

I think we should be able to safe-guard us for the future, and make the warning go away, by using `if #available(iOS 11.3, *)` 🤔 

Hopefully, I'll have time to look into a PR later today ☺️

--- a/Sources/SecureEnclaveAccessControl.swift
+++ b/Sources/SecureEnclaveAccessControl.swift
@@ -76,14 +76,14 @@ public enum SecureEnclaveAccessControl: Int, CustomStringConvertible, Equatable 
             return .userPresence
         case .biometricAny:
             if #available(macOS 10.12.1, *) {
-                return .touchIDAny
+                return .init(rawValue: 2) // .biometryAny with Xcode 9 compatibility.
             } else {
                 ErrorHandler.assertionFailure(".biometricAny requires macOS 10.12.1.")
                 return .userPresence
             }
         case .biometricCurrentSet:
             if #available(macOS 10.12.1, *) {
-                return .touchIDCurrentSet
+                return .init(rawValue: 8) // .biometryCurrentSet with Xcode 9 compatibility.
             } else {
                 ErrorHandler.assertionFailure(".biometricCurrentSet requires macOS 10.12.1.")
                 return .userPresence

